docs/program/apis/sessions.md

```diff
@@ -23,7 +23,7 @@ The period of time during which a client is connected to a robot is called a _se
 
 _Session management_ is a presence mechanism that allows you to manage the clients that are authenticated and communicating with a robot's `viam-server` instance.
 
-As a safety precaution, the default session management configuration ensures that a robot only moves when a client is actively connected.
+It also functions as a safety precaution: the default session management configuration ensures that a robot only moves when a client is actively connected and stops any components that remain running when a client disconnects.
```

Comment:
I would say that safety is the _primary_ function of the session manager. it doesn't exist for any other reason at the moment.